test/jdk/tools/launcher/SourceMode.java

```diff
@@ -26,7 +26,7 @@
  * @bug 8192920 8204588 8210275 8286571
  * @summary Test source mode
  * @modules jdk.compiler jdk.jlink
- * @run main SourceMode
+ * @run main/othervm SourceMode
```

Comment:
This looks fine, might need to add `@comment` to something brief to say why the test are othervm. Also I assume you'll bump the copyright header on the tests before ingtegrating.

Reply:
Thanks. `@comment` added and copyright header updated.